packages/ember-glimmer/lib/helpers/action.js

```diff
@@ -57,14 +64,15 @@ export class ClosureActionReference extends CachedReference {
       if (!action) {
         throw new EmberError(`An action named '${actionName}' was not found in ${target}`);
       }
+    } else if (action && typeof action[INVOKE] === 'function') {
+      target = action;
+      action = action[INVOKE];
     } else if (actionType !== 'function') {
-      throw new EmberError(`An action could not be made for \`${rawAction}\` in ${target}. Please confirm that you are using either a quoted action name (i.e. \`(action '${rawAction}')\`) or a function available in ${target}.`);
+      // TODO: Is there a better way of doing this?
```

Comment:
This actually was in reference to the line below the todo: 
`let rawActionLabel = rawActionRef._propertyKey || rawAction;`

There's no guarantee, I think, that rawActionRef is a propertyReference, and accessing a private property on the reference anyway is problematic.